keyboards/themadnoodle/noodlepad_micro/info.json

```
@@ -0,0 +1,70 @@
+{
+    "manufacturer": "The Mad Noodle",
+    "keyboard_name": "NoodlePad Micro",
+    "maintainer": "the-mad-noodle",
+    "url": "https://www.madnoodleprototypes.com/",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "hue_steps": 10,
+        "led_count": 4,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP0"],
+        "rows": ["GP26", "GP27", "GP28"]
+    },
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "3.0.0",
+        "pid": "0x0004",
+        "vid": "0x6A6C"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x3": {
+            "layout": [
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [1, 2], "x": 2, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [2, 2], "x": 2, "y": 2},
+                {"matrix": [2, 1], "x": 1, "y": 2},
+                {"matrix": [2, 0], "x": 0, "y": 2}
+            ]
+        }
+    },
+
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            { "pin_a": "GP2", "pin_b": "GP1", "resolution": 4 }
+            { "pin_a": "GP3", "pin_b": "GP4", "resolution": 4 }
```

Should not duplicate default settings
```suggestion
            { "pin_a": "GP2", "pin_b": "GP1" }
            { "pin_a": "GP3", "pin_b": "GP4" }
```